package.json

```diff
@@ -51,7 +51,6 @@
     "rules": {
       "react/forbid-prop-types": "off"
     },
-    "parser": "babel-eslint",
     "settings": {
```

Comment:
Can we remove these settings as well? These are set in the default config as well: https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/eslint.config.js#L43

Reply:
https://github.com/cerner/terra-framework/pull/1207/commits/5611ac081c53b9c1764e25aa6a8ce09632bd7838